client.go

```diff
@@ -705,10 +706,21 @@ func (c *Client) Deploy(ctx context.Context, path string) (err error) {
 	// Deploy a new or Update the previously-deployed Function
 	c.progressListener.Increment("Deploying function to the cluster")
 	result, err := c.deployer.Deploy(ctx, f)
+
+	var namespace string
+	if f.Namespace == "" {
+		var err1 error
+		namespace, err1 = k8s.GetNamespace("")
+		if err1 != nil {
+			return err1
+		}
+	} else {
+		namespace = f.Namespace
+	}
 	if result.Status == Deployed {
-		c.progressListener.Increment(fmt.Sprintf("Function deployed at URL: %v", result.URL))
+		c.progressListener.Increment(fmt.Sprintf("Function deployed in namespace %q and exposed at URL: %v", namespace, result.URL))
 	} else if result.Status == Updated {
-		c.progressListener.Increment(fmt.Sprintf("Function updated at URL: %v", result.URL))
+		c.progressListener.Increment(fmt.Sprintf("Function updated in namespace %q and exposed at URL: %v", namespace, result.URL))
```

Comment:
```suggestion
		c.progressListener.Increment(fmt.Sprintf("Function updated in namespace %q and exposed at URL: \n%v", namespace, result.URL))
```
See the comment: https://github.com/knative-sandbox/kn-plugin-func/issues/1083#issuecomment-1169739020